logstash-core/lib/logstash/config/source/local.rb

```diff
@@ -102,7 +102,7 @@ def get_unmatched_files
       end
 
       def valid_encoding?(content)
-        content.ascii_only? && content.valid_encoding?
+        content.ascii_only? || content.valid_encoding?
```

Comment:
is there a situation where the encoding is valid but it's not ascii, or it is ascii but not valid? maybe `content.valid_encoding?` is enough.

Also, I suggest double checking if this has any interference with the `config.support_escapes` feature ( don't think it does, just worth checking)